README.md

```diff
@@ -63,4 +63,4 @@ Note: The localhost client is currently non-functional.
 
 - [IBC Website](https://ibcprotocol.org/)
 - [IBC Specification](https://github.com/cosmos/ibc)
-- [Documentation](docs/README.md)
+- [Documentation](http://ibc.cosmos.network/main/ibc/overview.html)
```

Comment:
Can you put this PR on hold for a couple of days? Marko was trying to setup https.

Reply:
https://ibc.cosmos.network is working now.

Reply:
```suggestion
- [Documentation](https://ibc.cosmos.network/main/ibc/overview.html)
```